client/state/selectors/test/get-jetpack-remote-install-error.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { getJetpackRemoteInstallError } from 'state/selectors';
+
+const url = 'https://yourgroovydomain.com';
+
+describe( 'getJetpackRemoteInstallError()', () => {
+	test( 'should return null if no errors', () => {
+		const state = {
+			jetpackRemoteInstall: {
+				error: {},
+			},
+		};
+		expect( getJetpackRemoteInstallError( state, url ) ).toBe( null );
```

Comment:
[`toBeNull()`](https://facebook.github.io/jest/docs/en/expect.html#tobenull) is preferred (although it just provides nicer error messages.